spec/v1/CSN-Interop-Effective.schema.yaml

```diff
@@ -167,6 +167,26 @@ definitions:
     description: |-
       Metadata that describes the document (and what it represents) itself.
     properties:
+      name:
+        type: string
+        maxLength: 255
+        description: |-
+          Machine readable technical name / local ID of the CSN document.
```

Comment:
```suggestion
      localId:
        type: string
        maxLength: 255
        description: |-
          Machine readable local ID of the CSN document. 
          The ID is local to the source system only. In combination with the  `meta.document.namespace` and version it becomes globally unique.
```

Reply:
We voted in CSN Interop workstream and decided to stick to `name`
FYI, @maiargu